tests/test_integration.py

```diff
@@ -0,0 +1,138 @@
+"""
+Integration tests for depdag.
+"""
+import unittest
+from depdag import DepDag
+
+
+class TestInegration(unittest.TestCase):
+
+    def test_case_1(self):
+        vert = DepDag().vertices
+        vert.a.depends_on('b', 'c')
+        vert.b
```

Comment:
Vertex named `b` has been created at line 12, so we can skip this one and I suggest we do if you don't mind.